src/components/Dialogs/AdvancedDelegateDialog/AdvancedDelegateDialog.tsx

```diff
@@ -56,37 +50,26 @@ export function AdvancedDelegateDialog({
   const [directDelegatedVP, setDirectDelegatedVP] = useState<bigint>(0n);
   const { setOpen } = useModal();
   const params = useParams<{ addressOrENSName: string }>();
-  const { contracts } = Tenant.current();
-
-  const getOpBalance = async (address: `0x${string}`) => {
-    return await contracts.token.contract.balanceOf(address);
-  };
+  const { contracts, slug } = Tenant.current();
 
   const fetchData = useCallback(async () => {
     try {
       if (!address) return;
-      const promises = [
-        // TODO temporary fetch all query - optimization via API needed
-        fetchAllForAdvancedDelegation(address),
-        getOpBalance(address),
-      ];
-
-      const [getAll, delegateOpBalance] = await Promise.all(promises);
 
       // @ts-ignore
       const [
-        balance,
+        subdelegateBalance,
         isDelegating,
         delegatees,
         proxyAddress,
         delegatorsRes,
         directDelegatedVP,
-      ] = getAll;
+      ] = await fetchAllForAdvancedDelegation(address)
 
       setDirectDelegatedVP(directDelegatedVP);
-      setAvailableBalance(balance);
+      setAvailableBalance((BigInt(subdelegateBalance) + directDelegatedVP).toString());
```

Comment:
This is the issue. The `subdelegateBalance` balance was zero overriding the token balance.